### PR TITLE
New, refreshed website

### PR DESCRIPTION
**Pull request by DominikSerafin**
Hey @djfarrelly, on top of my other changes I've decided to refresh MailDev website as well. The original one is a bit outdated and has some issues, like e.g. horizontal scrollbars. I also wanted to learn & play with Tailwind a bit, so this was a perfect opportunity.

I hope you'll like it! If you do - please let me know and I'll merge this PR. Or of course, you can also do it yourself.

The new website can be previewed at https://github.serafin.io/maildev/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,277 +1,558 @@
...
+
+    t._e = [];
+    t.ready = function(f) {
+      t._e.push(f);
+    };
+
+    return t;
+  }(document, "script", "twitter-wjs"));</script>
+
+  <script src="https://static.buffer.com/js/button.js"></script>
```
Comment (djfarrelly) [inline index.html:521]:
```suggestion
```

We can remove this one

File: index.html [hunk 2]
```diff
@@ -1,277 +1,558 @@
...
+
+  <footer>
+
+  </footer>
+
+
+
+  <footer class="bg-brand py-12  text-center">
+    <div class="container max-w-screen-md mx-auto px-4">
+      <p class="text-white font-medium">This project is maintained by <a class="underline font-semibold" href="http://danfarrelly.nyc">Dan Farrelly</a></p>
```
Comment (djfarrelly) [inline index.html:496]:
```suggestion
```

@DominikSerafin, we can remove this line too. With it being moved to the `maildev` github org and folks like you making the major contributions, this feels outdated 😁

**Review by djfarrelly — APPROVED** [review]
This looks awesome @DominikSerafin!! I love the way it looks and how you laid out the features. The hover to preview dark mode on the screenshot is a cool touch 🙌 

Nice work and thank you!

**Comment by DominikSerafin** [comment]
@djfarrelly I'm really happy that you like it! I've now also introduced your requested changes.

**Comment by soulteary** [comment]
I am cleaning up accumulated pr, found this amazing commit.  @DominikSerafin 

I love the new visuals and details as much as @djfarrelly  does, I will merge it and submit some content tweaks later.

Perhaps in the future, we could have a script to automatically update the content of the document to the website.

**Comment by soulteary** [comment]
I have updated the website content with the content of the current project https://github.com/maildev/maildev/pull/386

**Pull request merged**